Documentation/ApiOverview/RequestHandling/Index.rst

```diff
@@ -0,0 +1,272 @@
+.. include:: ../../Includes.txt
+
+.. highlight:: php
+
+.. _request-handling:
+
+================
+Request Handling
```

Comment:
I would consider using psr-15 or middlewares in the title because these terms have been used in the changelog and this is what people may be looking for. e.g. "Request Handling (Middlewares)" or "Request Handling (PSR-15)"

Reply:
The Idea was to document the whole request handling, where Middlewares are just one part. If we  make headlines and file names (url) to specific, we would need to rename them afterwards.

Reply:
ok. Thanks for clearing that up :thumbsup: